test/demo/recreate.inttest.ts

```diff
@@ -0,0 +1,144 @@
+            const notes = await notifsRemote.getAll(testWindow.identity);
+            expect(notes.some(note => note.id === firstOptions.id)).not.toBeTruthy();
+
+            // Create the "exisitng" notificaiton
+            existingNote = await notifsRemote.create(testWindow.identity, firstOptions);
+        });
+
+        afterEach(async () => {
+            await notifsRemote.clearAll(testWindow.identity);
+            await testApp.quit(true).catch(() => {});
+        });
+
+        test('The promise resolves to the new notification object', async () => {
+            const newNotePromise = notifsRemote.create(testWindow.identity, secondOptions);
+            expect(newNotePromise).resolves;
+
+            expect(await newNotePromise).toMatchObject(secondOptions);
+        });
+
+        test('No "cleared" event is emitted', async () => {
+            // Register a closed listener on the window
+            const closeListener = jest.fn<void, [NotificationClosedEvent]>();
+            await notifsRemote.addEventListener(testWindow.identity, 'notification-closed', closeListener);
+
+            // Recreate the notification and pause momentarily to allow the event to propagate
+            await notifsRemote.create(testWindow.identity, secondOptions);
+            await delay(100);
+
+            // Listener not triggered
+            expect(closeListener).not.toHaveBeenCalled();
+        });
+
+        test('The card in the center is updated to show the new notification details', async () => {
+            // Existing card matches `firstOptions`
+            await assertDOMMatches(CardType.CENTER, testApp.identity.uuid, existingNote);
+
+            // Recreate the notification
+            const newNote = await notifsRemote.create(testWindow.identity, secondOptions);
+
+            // New card matches `secondOptions`
+            await assertDOMMatches(CardType.CENTER, testApp.identity.uuid, newNote);
+        });
+
+        // Extra tests for toasts only when the center is hidden
+        if (!showCenter) {
+            describe('When the existing notification has an active toast', () => {
+                test('The existing toast window is closed', async () => {
+                    const expectedEvent = {
+                        topic: 'system',
+                        type: 'window-closed',
+                        ...getToastIdentity(testApp.identity.uuid, firstOptions.id!)
+                    };
+
+                    // Listen for window-closed events globally
+                    const finCloseListener = jest.fn<void, [WindowEvent<'system', 'window-closed'>]>();
+                    await fin.System.addListener('window-closed', finCloseListener);
+
+                    // Recreate the notification and pause momentarily to allow the service time to process
+                    await notifsRemote.create(testWindow.identity, secondOptions);
+                    await delay(200);
```

Comment:
Make consistent with the other 'waiting for events' delays following notifsRemote.create?